docs/root/configuration/network_filters/mongo_proxy_filter.rst

```diff
@@ -147,10 +147,10 @@ mongo.mongo.drain_close_enabled
   % of connections that will be drain closed if the server is draining and would otherwise
   attempt a drain close. Defaults to 100.
 
-mongo.fault.fixed_delay.percent
+mongo.fault.fixed_delay.percentage
```

Comment:
Did this key name change? I don't think so? It shouldn't?

Reply:
Oops, that's correct. Addressed in fa53068.